src/tokens.js

```diff
@@ -422,6 +505,10 @@ const tokens = {
       outputs.push({'address': newAddress, 'value': TOKEN_MELT_MASK, 'tokenData': AUTHORITY_TOKEN_DATA});
     }
 
+    // withdraw HTR tokens
+    const withdrawAmount = helpers.getDepositAmount(amount);
```

Comment:
Isn't it supposed to be `getWithdrawAmount`?

Reply:
Thanks!!